src/Run/Setup.cs

```diff
@@ -14,13 +18,51 @@ public class Setup
         public string Os { get; set; }
         public string ConfigurationFilePath { get; set; }
 
+        private string ParseSettingValue(string inputValue)
+        {
+            string value = string.Empty;
+            int length = inputValue.Length;
+            for (int i = 0; i < length; i++)
+            {
+                if (inputValue[i] == '$')
```

Comment:
We could just ask if the first character of the string is `$`. If it is keep processing, otherwise just return the inputValue